### DISCUSSION TITLE
Image URL in the card block

### DISCUSSION THREAD

**Discussion by SantoshSrinivas79** [Budibase/budibase#3714] [Help] [answered]
Could you pls let me know how to link the Card block image to an image file?
The table shows the image but not the card [here](https://wands.notion.lol/gpt#/tools)

Thank you!

**Comment by mjashanks** [comment, marked as answer]
Ah yes - something has changed around attachments.

You need to use something like: `{{ New Cards block.Links.Logo.0.url }}` for your binding..... the key bit is `.0.url`

Attachments are actually an array (list) of JSON objects, that looks a bit like

```json
[
   { "name": "My-file-name.jpg", "url": "https://yourdomain.com/path/to/image.jpg"  }
]
```

So
- `Logo.0` - the first item in the list
- `0.url` - the url property

**Comment by SantoshSrinivas79** [comment]
Thanks @mjashanks ... looks good ... would be nice if we have 3 options for the card size ... similar to notion.so
